Website/src/rendering/src/pages/_app.tsx

```diff
@@ -44,28 +47,30 @@ function App({ Component, pageProps, router }: AppProps): JSX.Element {
 
   // DEMO TEAM CUSTOMIZATION - Add head section and CDP integration
   return (
-    <>
-      <Head>
-        <meta charSet="UTF-8"></meta>
-        <meta httpEquiv="X-UA-Compatible" content="IE=edge"></meta>
-        <meta name="viewport" content="width=device-width, initial-scale=1.0"></meta>
-        <meta name="description" content="Play! Summit" />
-      </Head>
+    <Provider store={reduxStore}>
```

Comment:
This file is from the JSS-Next.js boilerplate. We try to modify this file as least as possible and identify changes to facilitate upgrades to future JSS releases.

Is it necessary for the Redux provider and `OcProvider` to wrap everything including the `<Head>` section, CDP and Discover script tags? Could they wrap only `<Component {...rest} />`? If it is possible, it would be the preferred minimal changes to this JSS boilerplate file. The change should also be wrapped with a comment block:

```jsx
{/* DEMO TEAM CUSTOMIZATION - OrderCloud integration */}
<Provider store={reduxStore}>
  <OcProvider>
    <Component {...rest} />
  </OcProvider>
</Provider>
{/* END CUSTOMIZATION */}
```

Reply:
makes sense. updated